temba/orgs/models.py

```diff
@@ -86,6 +86,9 @@
 ACCOUNT_SID = 'ACCOUNT_SID'
 ACCOUNT_TOKEN = 'ACCOUNT_TOKEN'
 
+TWILIO_COMPLIANT_API_ACCOUNT_SID = 'TWILIO_COMPLIANT_API_ACCOUNT_SID'
+TWILIO_COMPLIANT_API_ACCOUNT_TOKEN = 'TWILIO_COMPLIANT_API_ACCOUNT_TOKEN'
```

Comment:
No there's no searching for available numbers or anything. In fact the Twilio REST Client doesn't need to be used at all for setting up the channel. The webhook URL https://github.com/dwilkie/rapidpro/blob/55487f22f0915dad4dfb4391d72c198dd547d65b/temba/orgs/models.py#L616 needs to be displayed somewhere in the channel configuration and set up manually inside Adhearsion